### PR TITLE
#668 Quick fix - Showing "Assigned to you" wrong label 

### LINKED ISSUE #668: Fix assignment rows showing "Assigned to you" in the wrong case

### PR DESCRIPTION
**Pull request by nmadruga**
Super quick bug fix! Fixes #668.

Also organised the file a little to be better to read.

### CODE REVIEW

**Comment by CarlosNZ** [comment]
I can't see any description of what the bug is, exactly, on the original issue.

I'm happy to approve since it's just a quick fix, but it'd be good to quickly test, so can you explain how to reproduce the bug if you have a sec? Thanks.

File: src/components/Review/ReviewSectionRowAssigned.tsx [hunk 1]
```diff
@@ -1,72 +1,93 @@
...
+        <ReviewerLabel {...reviewer} />
+      </>
+    }
+  />
+)
+
+const CurrentReviewLockedLabel: React.FC = () => (
+  <Label
+    className="simple-label"
+    icon={<Icon name="ban" size="small" color="red" />}
```
Comment (CarlosNZ) [inline src/components/Review/ReviewSectionRowAssigned.tsx:83]: `color="pink"`?

File: src/components/Review/ReviewSectionRowAssigned.tsx [hunk 2]
```diff
@@ -1,72 +1,93 @@
...
-  )
+  const getLabel = () => {
+    switch (action) {
+      case ReviewAction.canSelfAssign:
+        return isAssignedToCurrentUser ? (
+          <CurrentSelfAssignmentLabel />
+        ) : (
+          <TheirSelfAssignmentLabel {...assignment.reviewer} />
+        )
+      case ReviewAction.canStartReview:
```
Comment (andreievg) [inline src/components/Review/ReviewSectionRowAssigned.tsx:20]: should this also include `canReReview` and `canUpdate` ?

**Review by andreievg — COMMENTED** [review]
Looks pretty good, although i didn't look at previous logic (it's hard for be to deduce nested `?` `:` logic)

File: src/components/Review/ReviewSectionRowAssigned.tsx (hunk 2 above)
Comment (nmadruga) [reply src/components/Review/ReviewSectionRowAssigned.tsx:20]: Yes, sure thing

**Comment by nmadruga** [comment]
Thanks for review, I ended adding another fix - and a little different logic: Now it also display labels for "Can be self-assigned" and "Self Assignment locked" to others. Better than the empty row that was showing previously.

I think there are a few things to improve, for example after the review is done by you it still show "Assigned to you". We should probably show something as "Reviewed by you" at this point. But, not for this PR... I'll make a separated issue for that one.

File: src/utils/helpers/structure/generateReviewSectionActions.ts [hunk 3]
```diff
@@ -100,6 +104,8 @@ const generateReviewSectionActions: GenerateSectionActions = ({
       isReviewActive: (section.reviewAndConsolidationProgress?.totalActive || 0) > 0,
     }
 
+    console.log(isReviewable, isAssignedToCurrentUser, checkMethodProps)
```
Comment (andreievg) [inline src/utils/helpers/structure/generateReviewSectionActions.ts:107]: stray log

**Review by andreievg — APPROVED** [review]
Looks good, there is a console log though.

**Pull request merged**